### PR TITLE
Length 3 pointsets

### PR DESCRIPTION
**Pull request by mattsears18**

### CODE REVIEW

File: src/hull.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ function hull(pointset, concavity, format) {
         maxEdgeLen = concavity || 20;
 
     if (pointset.length < 4) {
-        return pointset.slice();
+        return pointset.concat([pointset[0]]).slice();
```
Comment (andriiheonia) [inline src/hull.js]: concat() already returns a new array so slice() call can be removed from here. Please don't forget to regenerate dist/hull.js after this change.

**Review by andriiheonia — CHANGES_REQUESTED** [review]
Thanks for the pull request and sorry for the delay with the review. I'd kindly ask you also to update the Changelog section in README.md. I also commented on one line in src/hull.js please take a look.

File: src/hull.js (hunk 1 above)
Comment (mattsears18) [reply src/hull.js]: I removed the slice() and also checked the point count after filtering duplicates.

**Comment by andriiheonia** [comment]
Looks good. Thanks!

**Pull request merged**